src/js/render.js

```diff
@@ -225,6 +226,10 @@ function Renderer() {
             return text;
         },
 
+        replaceParagraphs: function (text) {
+            return text.replace(/\\p[^a-z]/g, "\n");
```

Comment:
I think you probably want "not word" here rather than "[^a-z]".  For example, "this \pPhrase" doesn't render properly because of the capital P.  I recommend you try using `\\p\W` instead.